src/Cache/FileCache.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Cdek\Cache;
+
+use Cdek\Exceptions\CdekApiException;
+use Cdek\Loader;
+
+class FileCache
+{
+    const CACHE_FILE_NAME = '.cache.php';
+    private static array $store;
+
+    public static function getVars(): ?array
+    {
+        if(!file_exists(Loader::getPluginPath() . DIRECTORY_SEPARATOR . self::CACHE_FILE_NAME)){
+            return null;
+        }
+
+        return self::$store[self::CACHE_FILE_NAME] ?? self::$store[self::CACHE_FILE_NAME] = require_once(Loader::getPluginPath() . DIRECTORY_SEPARATOR . self::CACHE_FILE_NAME);
+    }
+
+    /**
+     * @param array|null $vars
+     *
+     * @return void
+     * @throws CdekApiException
+     */
+    public static function putVars(?array $vars): void
+    {
+        if(empty($vars)){
```

Comment:
```suggestion
        if($vars === null){
```

Такая проверка в данном случае экономичнее